.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -20,23 +20,44 @@ body:
     attributes:
       value: |
         ## Required information
+
   - type: textarea
     id: description
     attributes:
       label: Description
-      description: Please provide a clear, concise and descriptive explanation of what the bug is. Include screenshots or a video if needed. Tell us what were you expecting to happen instead of what is happening now.
+      description: | 
+        Please provide a clear, concise and descriptive explanation of what the bug is. 
+        Make sure you tell us:
+        - [ ] What is the **actual** behavior
+        - [ ] What is the **expected** behavior
+      value: |
+        ### What happens? 
+
+        ___
+        <details>
+        ```
+        If you have long snippets of code or logs, place them HERE
+        ```
+        </details>
     validations:
       required: true
+    id: screenshot
+    attributes:
+      label: Screenshot/Recording
+      description: If possible, please add screenshots or recording. 🙏
```

Comment:
We don't use emojis anywhere, why should we use it here?

Reply:
To get some attention. I think we should try to force people to read the description. Sometimes we get bug descriptions, that consist of error log without any explanation

Reply:
I like the emoji but it should be without the trailing dot on the sentence